xapian-applications/omega/index_file.cc

```diff
@@ -673,8 +692,9 @@ index_mimetype(const string & file, const string & urlterm, const string & url,
 
     // if the ctime has changed but the mtime is unchanged, we can just
     // update the existing Document and avoid having to re-extract text, etc.
+    string md5 = "";
```

Comment:
If you just write `string md5;` the string is empty by default - there's no need to assign '""` to achieve that, and at least with with some compilers it's a little more efficient not to.